Web_Development/Javascript/Topics/5.Syntax/2.Operators/README.md

```diff
@@ -0,0 +1,102 @@
+|a >>= b  | a = a >> b   |Assigns the results of a shifted right side by b to a |
+|a &= b   | a = a & b    |Assigns the results of a AND b to a                   |
+| a \|= b | a = a \| b   |Assigns the results of a OR b to a                    |     
+|a ^= b   | a = a ^ b    |Assigns the results of a XOR b to a                   |
+
+
+
+### [Arithmetic Operators](ArithmeticOperator.md#arithmetic-operators)
+An Arithmetic operators are used to perform arithemetical operations on given values.
+
+| Operator              | Example                                                     | Output        |
+|-----------------------|-------------------------------------------------------------|---------------|
+| Plus (+)              | `console.log(1+3);`                                         | `4`           |
+| Minus (-)             | `console.log(1-3);`                                         | `-2`          |
+| Multiplication (*)    | `console.log(4*3);`                                         | `12`          |
+| Division (/)          | `console.log(12/3);`                                        | `4`           |
+| Modulus (%)           | `console.log(12%3);`                                        | `2`           |
+| Exponent (^)          | `console.log(2^3);`                                         | `8`           |
+| Post-Increment (++)   | `let a = 5;`<br>`console.log(a++);`<br>`console.log(a);`     | `5`<br>`6`    |
+| Post-Decrement (--)   | `let a = 5;`<br>`console.log(a--);`<br>`console.log(a);`     | `5`<br>`4`    |
+| Pre-Increment (++)    | `let a = 5;`<br>`console.log(++a);`<br>`console.log(a);`     | `6`<br>`6`    |
+| Pre-Decrement (--)    | `let a = 5;`<br>`console.log(--a);`<br>`console.log(a);`     | `4`<br>`4`    |
+
+
+
+### [Comparison Operators](ComparisonOperator.md#comparison-operators)
+Comparison operators are used to compare values and return true or false based on the result.
+
+| Operator                  |  Example                                                     | Output        |
+|---------------------------|--------------------------------------------------------------|---------------|
+| Equal (=)                 | `console.log(1 == 2);`                                       | `false`       |
+| Not Equal (!=)            | `console.log(1 != 2);`                                       | `true`        |
+| Greater Than (>)          | `console.log(1 > 2);`                                        | `false`       |
+| Less Than (<)             | `console.log(1 < 2);`                                        | `true`        |
+| Greater Than or Equal (>=)| `console.log(1 >= 2);`                                       | `false`       |
+| Less Than or Equal (<=)   | `console.log(1 <= 2);`                                       | `true`        |
+| Strict Equality (===)     | `console.log(1 === '1');`                                    | `false`       |
+
```

Comment:
It should be `Strict Equal To (===)`, change this.
Also add `Strict Not Equal To (!==)` after this with an example.